### PR TITLE
modified the rllab setup

### PR DESCRIPTION
**Pull request by AboudyKreidieh**
they should work now for everyone that had isssue

### CODE REVIEW

File: docs/source/flow_setup.rst [hunk 1]
```diff
@@ -152,7 +150,7 @@ Getting started (rllab-multiagent)
 To run any of the RL examples, make sure to run
 ::
 
-    source activate flow
+    source activate flow-rllab
```
Comment (eugenevinitsky) [inline docs/source/flow_setup.rst]: This is not the case if you're not using rllab; you won't have this env.

Comment (AboudyKreidieh) [reply docs/source/flow_setup.rst]: u create the environment during the rllab setup instructions

Comment (AboudyKreidieh) [reply docs/source/flow_setup.rst]: this is in the Setting up section for rllab btw, which is totally optional and assumes u've installed rllab

Comment (eugenevinitsky) [reply docs/source/flow_setup.rst]: Mmm, I was confused by the git diff. Okay!

**Pull request merged**